test/ui-test/src/screens/StatusCommunityScreen.py

```diff
@@ -146,12 +149,28 @@ def create_community_category(self, community_category_name, community_channel_n
             test.fail("Can't find channel " + community_channel_name)
 
         click_obj_by_name(CreateOrEditCommunityCategoryPopup.COMMUNITY_CATEGORY_BUTTON.value)
+        
+    def delete_community_category(self, community_category_name):
+        [loaded, category] = self._find_category_in_chat(community_category_name)
+        if not loaded:
+            test.fail("Can't find category " + community_category_name)
```

Comment:
Nowadays we are using `SquishDriverVerification` and not direct calls to squish api. Could you replace it accordingly?